.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+name: Publish Ozone E2E Tests
+
+on:
+
+  workflow_dispatch:
```

Comment:
`workflow_dispatch` allows you to manually trigger a workflow run from GitHub's UI. For you would want to publish on push to the `main` and maybe on release. `workflow_dispatch` would allow you to manually trigger the publishing process at any time, not just when code is pushed to `main` or when a release is created.
```yaml
name: Publish to Mekom's NPM Registry.

on:
  push:
    branches:
      - main
  release:
    types:
      - created
  workflow_dispatch:
```

Reply:
Just drop the `workflow_dispatch` or do you have a use case for it?

Reply:
> or do you have a use case for it?
no,

Just need to publish when there is a push on main or release.